CustomTracks/Backgrounds/CustomBackground.cs

```diff
@@ -22,6 +23,65 @@ public CustomBackground(AssetBundle bundle, string songPath) : base(bundle)
     public override GameObject Load(BackgroundContext ctx)
     {
         var bg = Bundle.LoadAsset<GameObject>("assets/_background.prefab");
+
+        // MacOS Shader Handling
+        // May need to be expanded to other platforms eventually, but for now only check for invalid shaders on mac
+        if (Application.platform == RuntimePlatform.OSXPlayer)
+        {
```

Comment:
Think it might be beneficial to put all this stuff in a seperate function for readability purposes, since this block of code is a bit chunky